package.json

```
@@ -102,7 +102,7 @@
     "react-native-haptic-feedback": "2.2.0",
     "react-native-image-picker": "7.0.2",
     "react-native-keyboard-accessory": "0.1.16",
-    "react-native-keychain": "8.1.2",
+    "react-native-keychain": "git+ssh://git@github.com:synonymdev/react-native-keychain.git#refactor/android-data-store",
```

May want to switch to http for pulling the lib, so tests can run. I see the repo is publicly accessible

```suggestion
    "react-native-keychain": "github:synonymdev/react-native-keychain.git#refactor/android-data-store",
```

---

Ah that's right, nice catch. Updated [here](https://github.com/synonymdev/bitkit/pull/1611/commits/c8c4b4740f534b42cbc968382467c862ec78ec89).